core/frontend-devtools/README.md

```diff
@@ -130,7 +130,7 @@ This package provides several keyins to control the display of background maps,
 
 * `fdt attach maplayer <name>` - Attach a background map layer from name within the map layer source list.  Partial names may be used.
 * `fdt attach mapoverlay <name>` - Attach an overlay map layer from name within the map layer source list.  Partial names may be used.
-* `fdt attach model maplayer <name>` - Attach a model map layer for each unique model of the currently selected elements.
+* `fdt attach model maplayer <globe|reality> [name]` - Attach a model map layer for each unique model of the currently selected elements to either the globe or attached reality models. If name is specified, use that.
```

Comment:
Key-ins are the primary API of this package, don't break them unnecessarily (e.g., make new arguments optional and append them).

Reply:
I changed the key-in as such, and pushed the change:

* `fdt attach model maplayer [name] [globe|reality]` - Attach a model map layer for each unique model of the currently selected elements to either the globe or attached reality models. If `globe` or `reality` is not specified, the attachment will default to `globe`. If `name` is specified, use that. If `name` is specified as `0`, that is considered `undefined` or unspecified.